ext/dev.js

```diff
@@ -117,10 +117,14 @@
 			}
 		},
 		_logger:function(type, arr){
-			if(console.log.apply){
-				console[type].apply(console, arr);
-			} else {
+			if (typeof console.log == "object") {
```

Comment:
Can you explain though?  If `type` is "log" and you are saying that console.log is an object, you can't call an object, it should throw. I don't doubt that your code is correct, I just want to understand it.